protobuf/flow/executiondata/executiondata.proto

```diff
@@ -15,6 +18,12 @@ service ExecutionDataAPI {
 
   rpc GetExecutionDataByBlockID(GetExecutionDataByBlockIDRequest)
       returns (GetExecutionDataByBlockIDResponse);
+
+  rpc SubscribeExecutionData(SubscribeExecutionDataRequest)
+      returns (stream SubscribeExecutionDataResponse);
+
+  rpc SubscribeEvents(SubscribeEventsRequest)
```

Comment:
A small API description + a note describing API and expected errors returned (e.g. what error to expect if block requested is across spork boundary.)

Also a tiny clarification on Subscribe events whether you are expected to get a response for every block or only ones that match the filter (latter may lead to RPC timeouts if there are prolonged periods w/o matched events.).